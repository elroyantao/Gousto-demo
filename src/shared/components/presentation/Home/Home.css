.Home {
  padding: 20px 10px;
  font-size: 20px;
}
