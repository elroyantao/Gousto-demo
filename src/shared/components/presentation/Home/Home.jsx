import React from 'react'
import Helmet from 'react-helmet'

export default function Home() {
  return (
    <div>
      <Helmet title="Gousto - Home" />
      <p> this is the home page</p>
    </div>
  )
}
