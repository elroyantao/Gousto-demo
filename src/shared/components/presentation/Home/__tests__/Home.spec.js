import testComponentHelper from '../../../../lib/test-helper'
import Home from '../Home'

describe('<Home/>', () => {
  const renderComponent = testComponentHelper(Home)
  describe('@renders', () => {
    it('in default state', () => {
      expect(renderComponent({}).getTree())
        .toMatchSnapshot()
    })
  })
})
