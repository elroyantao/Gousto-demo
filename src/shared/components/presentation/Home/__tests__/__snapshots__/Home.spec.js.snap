// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home/> @renders in default state 1`] = `
<div>
  <HelmetWrapper
    encodeSpecialCharacters={true}
    title="Gousto - Home"
  />
  <p>
     this is the home page
  </p>
</div>
`;
